### ISSUE TITLE
app is not accessible.

### ISSUE THREAD

**Issue by paxcoder** [mastodon/mastodon-ios#328]
I can't interact with toots using voiceover

**Comment by MainasuK** [comment]
Hi @paxcoder,

I'm sorry the interactions are not fully handled. The app only carry with basic toot reading a11y supports now. Recently, I've figure out how to response to the VoiceOver properly. So the most a11y issues should be fixed in the next version.

**Comment by TheQuinbox** [comment]
Ah, beat  me to it. Was just about to create this issue.

Most everything is accessible, it's just we can't do anything with toots because there are no VoiceOver actions, no buttons, and we can't click on the toot to bring it up in a different screen, even.

**Comment by paxcoder** [comment]
but it sucks.
we don't have any accessible app that is not abandonware.
it's why we need to use apps that was updated 3 years ago, or in the best case 9 months ago.

**Comment by paxcoder** [comment]
bump up, several months later, we are dumped. this app is abandonned too?

**Comment by MainasuK** [comment]
I know it's been a long time since I updated. But I don't have time to do it at the moment. I will try to get some updates after some busy with other projects.

And I wonder why VoiceOver not working? I testing the home timeline and it's should read the post content and doulbe tap will redirect to the converstaion page which allows you interacte with that. And I'm sorry for the missing VoiceOver actions. I cannot give any promise about update time. But that's will be fixed once I'm free from other projects.

**Comment by edhowland** [comment]
I am using version 1.5 of the IOS app on iPhone 12 mini, IOS version 16.3
The only noticable  VoiceOver problem I am currently having is the scrolling.

Turn on VoiceOver.
Go to almost any list or thread.
When you swipe right to see the next post and you reach the bottom of the curren page on screen it leaves the thread and does not automatically scroll down a page. 
It leaves the threads and jumps down to the tab bar.
You can work around this by going over to the scroll bar and swiping down one time and then going back over to thethread list.
You start at the top on page 2 and go to the bottom of page 2 where the same behaviour happens again.

Let me know if I can provide any other information.
